### ISSUE TITLE
MSS Agent - Add capability to sign iOS with entitlements

### ISSUE THREAD

**Issue by ShellyXueHan** [BCDevOps/developer-experience#1476]
**Describe the issue**

BCVV is asking for adding associated domains in the entitlement. This will have to be included as part of the signing process.

`codesign -f -s "${signingIdentifier}" --entitlements ./Entitlements.plist Payload/*.app`

**Definition of done**
- [ ] identify the best way of doing it
- [ ] implement

**Comment by jleach** [comment]
Looking into this. I was able to do this all through Xcode using the sample "Test & Sign" app we have. I just added the associated domains via Xcode, added a sample domain, and then went through the normal build, sign and upload process. I'll try a few other things this AM but I think it should be working out-of-the-box for teams. Maybe they don't know how to setup it up properly?


![Screen Shot 2021-08-16 at 10.39.17 AM.png](https://images.zenhubusercontent.com/5c2fa1b7ce4f66381cd8d02d/98471afc-1fbf-4839-879b-b4530337214a)
